lib/beacon_web/controllers/api/component_json.ex

```diff
@@ -0,0 +1,40 @@
+defmodule BeaconWeb.API.ComponentJSON do
+  alias Beacon.Content.Component
+
+  def index(%{component_definitions: definitions}) do
+    %{
+      menuCategories: [
```

Comment:
Usually Phoenix APIs use the snake case pattern since that's the pattern used in the code base on the backend side. Is it okay if we change it to?

```suggestion
      menu_categories: [
```

Reply:
Lets leave it camel cased and try to use a plug to convert between camelCase and snake_case transparently.